src/install.c

```diff
@@ -1159,6 +1159,85 @@ static gboolean launch_and_wait_default_handler(RaucInstallArgs *args, gchar* bu
 	return res;
 }
 
+static gchar* get_uptime(void)
+{
+	gchar *contents = NULL;
+	g_autoptr(GError) ierror = NULL;
+
+	if (!g_file_get_contents("/proc/uptime", &contents, NULL, &ierror)) {
```

Comment:
`/proc/uptime` contains two values, see `proc(5)`:
```
/proc/uptime
       This  file  contains  two numbers (values in seconds): the uptime of the
       system (including time spent in suspend) and the amount of time spent in
       the idle process.
```
Did you intentionally use both?

Reply:
Not intentionally. Will limit to using first one only